### PR TITLE
Linking pages

### PR DESCRIPTION
**Pull request by lcegit**
Pt Link: https://www.pivotaltracker.com/story/show/153510026

### CODE REVIEW

**Comment by lcegit** [comment]
Requesting to merge this PR, please let us know if there's any refactoring we can do in terms of structuring the seeds file? Thanks! @tochman

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -9,20 +9,24 @@
...
     <section id="content">
-      <% if notice %>
-        <p><%= notice %></p>
-        <% elsif alert %>
-          <p><%= alert %></p>
-        <% end %>
-        <%= yield %>
-      </section>
+      <% flash.each do |key, value| %>
+          <%= content_tag :div, value, class: "flash #{key}" %>
```
Comment (tochman) [inline app/views/layouts/application.html.erb]: I can't see any css classes defined for `flash` or the keys.

File: app/views/welcome/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,11 @@
 <h1>iStory</h1>
 <p>An interactive journey</p>
 <p></p>
-<div class="tale"><h1><%= link_to 'Tale' %></h1>
-  <p>Lorem ipsum dolor sit amet</p>
+
+<div class="stories">
```
Comment (tochman) [inline app/views/welcome/index.html.erb]: I can't see any css class defined for `stories` If you don't need one, please use `id`rather than `class`

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (lcegit) [reply app/views/layouts/application.html.erb]: Planning to include a reference for this in upcoming PR: https://www.pivotaltracker.com/story/show/153539405

Comment (tochman) [reply app/views/layouts/application.html.erb]: Then add Rhode classes in that pr. Remove for now, thx.

**Review by tochman — COMMENTED** [review]
I still don't understand why you are giving sections id's or undefined classes. Can't see that you are using them in many cases. Please keep your code dry.

**Review by tochman — APPROVED** [review]

**Comment by lcegit** [comment]
Thanks! Will take your comments into account next time. In some cases we needed to make use of classes in order to select content in our acceptance tests.

**Pull request merged**